src/Application/Regulation/Command/Period/SavePeriodCommandHandler.php

```diff
@@ -18,6 +19,10 @@ public function __construct(
 
     public function __invoke(SavePeriodCommand $command): Period
     {
+        usort($command->applicableDays, function (string $d1, string $d2) {
+            return ApplicableDayEnum::getDayIndex($d1) <=> ApplicableDayEnum::getDayIndex($d2);
+        });
```

Comment:
Le niveau d'abstraction est notablement plus bas ici, ça vaudrait le coup de mettre ça dans une méthode de la commande ?

```suggestion
       $command->sortApplicableDays();
```